sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/VersioningTest.groovy

```diff
@@ -30,7 +30,7 @@ class VersioningTest extends APISpec {
     def setup() {
         blobName = generateBlobName()
         containerName = generateContainerName()
-        blobContainerClient = versionedBlobServiceClient.createBlobContainer(containerName)
+        blobContainerClient = primaryBlobServiceClient.createBlobContainer(containerName)
```

Comment:
This is from when you moved the blob versioning tests to the primary test account?

Reply:
Yeah since I made the primary account have versioning enabled